elliottlib/bzutil.py

```diff
@@ -621,6 +626,9 @@ def get_tracker_bugs(self, bug_ids: List, strict: bool = False, verbose: bool =
     def get_flaw_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = False):
         return [b for b in self.get_bugs(bug_ids, permissive=not strict, verbose=verbose) if b.is_flaw_bug()]
 
+    def filter_attached_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = False):
```

Comment:
I'm thinking this would be better off as a static method in JIRABug and BugzillaBug , since it's not interacting with the BugTracker functionality?

Reply:
This method needs to distinguish between different bug types, I think it is appropriate to put them in different bugtrackers, if move the method into the bug class then it will instantiate each bug object while this will only need a bug id.